calendar/Package.md

```diff
@@ -0,0 +1,20 @@
+Connects to Microsoft Outlook Calendar from Ballerina
+
+### Package Overview
+The `outlook.calendar` is a [Ballerina](https://ballerina.io/) connector for Outlook Calendar.
+This package provides the capability to easily access Outlook Calendar.
```

Comment:
This introduction should have some value. All come here know the above information.

Reply:
Yeah.I will change the introduction. Shall we change the Documentation guideline?
![image](https://user-images.githubusercontent.com/19235998/126935888-bf371d4b-1608-4814-b828-95d627879262.png)

Reply:
changed the Intro. Added a comment in the doc. https://docs.google.com/document/d/1WZg1ix9MpKDTQ_OOm3NVqhQ5L0VGNaUQ1LRmjp-SMO4/edit?disco=AAAANWCuYwk